08_project/runnerdave/README.md

```diff
@@ -0,0 +1,37 @@
+# Puppy server
+
+Storage of Puppies with two different storage methods.
+
+## Pre-requisites
+
+You need to have [Go](https://golang.org/) installed
+
+## Build and install
+
+```
+go install ./...
+```
+
+### Run installed version
+
+```
+$GOPATH/bin/puppy-server
```

Comment:
just `puppy-server` with a note, assuming `$GOPATH/bin` is on your path?

Reply:
fixed